### ISSUE TITLE
[TECH ED] Stretch: Consider Tracks

### ISSUE THREAD

**Issue by SallyMcGrath** [CodeYourFuture/Module-Data-Flows#2]
- [ ] Explore CYF Tracks
- [ ] Extend your learning with self study

If you are confident you have finished the course (i.e. you meet the Success criteria for all modules), and have spare time, you may want to consider self-guiding yourself through some of the Tracks available.

Do not start learning Tracks until you are finished with this course! It is more important that you understand what you know really well, than that you understand lots of things.

### Link to the coursework

See https://tracks.codeyourfuture.io/ for available tracks.

**Comment by illicitonion** [comment]
Closing in favour of https://github.com/CodeYourFuture/Module-Data-Flows/issues/51
